src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+abstract class Machine { //abstract class for next Machine-type classes

```

Comment:
Not fixed
![image](https://user-images.githubusercontent.com/20326858/198274494-f7a5f6f4-52d0-46ed-b69e-d274ba70b2cc.png)

Reply:
+